packages/babel-helper-module-transforms/src/index.js

```diff
@@ -236,6 +236,7 @@ function buildNamespaceReexport(metadata, namespace, loose) {
         Object.keys(NAMESPACE).forEach(function(key) {
           if (key === "default" || key === "__esModule") return;
           VERIFY_NAME_LIST;
+          if (key in EXPORTS && EXPORTS[key] === NAMESPACE[key]) return;
```

Comment:
Since this is technically not spec compliant, could you add a comment explaining the reason to do it?

Reply:
Done (directly above the template string)